app/code/Magento/LoginAsCustomer/Test/Mftf/ActionGroup/StorefrontAssertStickyLoginAsCustomerNotificationBannerActionGroup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertStickyLoginAsCustomerNotificationBannerActionGroup">
```

Comment:
Hi @konarshankar07. 

Please, rename the action group according to the standards. We use `Assert<area><name>ActionGroup` pattern.

I know, It's always confusing :)

You may refer to the 11.3.5 section of the [tech guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#113-functional-testing) to make sure you are using the correct naming convention. 

Thank you!